db/migrate/20221219210301_remove_institution_signature_from_collection.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveInstitutionSignatureFromCollection < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :collections, :institution_signature, :boolean
```

Comment:
So in a more perfectionist environment, you'd probably remove the add migration for this column instead of adding the column and then removing it.  But we're going to roll with it because we don't care that much.

Reply:
Makes sense, thanks